src/stores/Counter.ts

```diff
@@ -0,0 +1,15 @@
+import { types } from 'mobx-state-tree';
+
+export const Counter = types
+  .model('Counter', {
+    counter: 0,
```

Comment:
`types.number`?

Reply:
`0` is a shorthand for `types.optional(types.number, 0)`

https://github.com/mobxjs/mobx-state-tree#using-a-mst-type-at-design-time